services/adgs/config/adgs_search_config.yaml

```diff
@@ -12,87 +12,117 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-collections:
-  - id: s1_adgs_OPER_AUX_ECMWFD_PDMC
-    station: adgs
-    query:
-      productType: OPER_AUX_ECMWFD_PDMC
-    stac_extensions: [ "https://stac-extensions.github.io/eo/v1.0.0/schema.json", "https://stac-extensions.github.io/projection/v1.0.0/schema.json", "https://stac-extensions.github.io/view/v1.0.0/schema.json" ]
```

Comment:
Note: python yaml does some reformatting but this doesn't change the way the file is read afterwards.